src/rscminus/game/NetworkStream.java

```diff
@@ -30,14 +30,15 @@
 import java.nio.channels.SocketChannel;
 
 public class NetworkStream {
+    public static final int BUFFER_SIZE = 5000;
```

Comment:
:thinking: the ornox wanted you to do something with this I think

Reply:
NetworkStream changes will be addressed in a separate PR